Source/Core/Core/PowerPC/Jit64/Jit_Integer.cpp

```diff
@@ -52,7 +52,8 @@ void Jit64::GenerateOverflow()
   // rare).
   static const u8 ovtable[4] = {0, 0, XER_SO_MASK, XER_SO_MASK};
   MOVZX(32, 8, RSCRATCH, PPCSTATE(xer_so_ov));
-  MOV(8, R(RSCRATCH), MDisp(RSCRATCH, (u32)(u64)ovtable));
+  MOV(64, R(RSCRATCH2), ImmPtr(ovtable));
+  MOV(8, R(RSCRATCH), MRegSum(RSCRATCH, RSCRATCH2));
```

Comment:
This function is only used in this file, and it always ends with:
  if (inst.Rc)
    ComputeRC(gpr.R(d));
  gpr.UnlockAll();

So yes.